wdtk-datamodel/src/main/java/org/wikidata/wdtk/datamodel/helpers/Datamodel.java

```diff
@@ -397,7 +397,7 @@ public static QuantityValue makeQuantityValue(BigDecimal numericValue,
 	 * @return a {@link QuantityValue} corresponding to the input
 	 */
 	public static QuantityValue makeQuantityValue(BigDecimal numericValue, ItemIdValue unit) {
-		return factory.getQuantityValue(numericValue, unit.getIri());
+		return factory.getQuantityValue(numericValue, unit);
```

Comment:
To avoid this conversion to String one needs to update the DataObjectFactory interface too, hence the other changes.